rise-google-sheet.html

```diff
@@ -292,17 +292,15 @@
         this._startTimer();
       },
 
-      _handleNoNetwork: function( resp ) {
+      _handleErrorsWithCachedData: function( resp ) {
         var self = this;
 
         this.$.data.getItem( this._getDataKey(), function( cachedData ) {
           if ( cachedData ) {
             self._setResults( cachedData.data.results );
-
-            self.fire( "rise-google-sheet-response", cachedData.data );
-          } else {
-            self.fire( "rise-google-sheet-error", resp );
+            resp = Object.assign( resp, cachedData.data );
```

Comment:
@stulees that is right, we will send it so the widget can log the error but at the same time load the cached data. cheers